mergin/client.py

```diff
@@ -205,19 +205,26 @@ def _do_request(self, request):
         try:
             return self.opener.open(request)
         except urllib.error.HTTPError as e:
-            if e.headers.get("Content-Type", "") == "application/problem+json":
-                info = json.load(e)
-                err_detail = info.get("detail")
+            server_code = None
+            server_response = None
+            if e.headers.get("Content-Type", "") == "application/json":
+                server_response = json.load(e)
+                err_detail = server_response.get("detail")
```

Comment:
Is this a valid thing to do (i.e. shall we expect "detail" here)? Do we have some example where server returns `application/json` content type? I thought that if there's `detail`, then it will be marked as `application/problem+json` ?